src/power_grid_model/core/options.py

```diff
@@ -17,16 +17,21 @@ class OptionSetter:
     setter for options
     """
 
+    _value: Any
     _setter: Callable
 
     def __init__(self, setter):
         self._setter = setter
 
     def __set__(self, instance: "Options", value: Any):
         self._setter(instance.opt, value)
+        self._value = value
 
-    def __get__(self, instance, owner):
-        raise NotImplementedError("Cannot get option value!")
+    def __get__(self, instance, owner) -> Any:
+        """Return the value that was most recently set.
+
+        No guarantees are given on whether the value was set correctly."""
+        return self._value
```

Comment:
Why do we want this? The descriptor is a global object, it will not have the correct value.

Reply:
my reasoning was that rather than keeping a dict of all desired settings, you can read it from the options, which should contain those exact setting anyway. this keeps e.g. the amount of arguments low.

if you have any better proposal, i'm more than happy to implement that

Reply:
You mean the amount of argument in `calculate_power_flow`?

Reply:
yes but also in `_calculate_impl`

Reply:
We don't have any option related stuff in `_calculate_impl` right? Just a single argument `Options`. I still do not get it why do we need a getter here.

Reply:
I think `symmetric` is too fundamental and you need to pass it as a separate argument in `_calculate_impl`

Reply:
fixed